dask/dataframe/io.py

```diff
@@ -450,6 +450,12 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
     if '*' in key:
         single_node = False
 
+    if 'format' in kwargs:
```

Comment:
"Those lines"?
You mean the two `if`s we're commenting on? they're still here, i just squashed the many commits this ended up with into one.